src/DynamoCoreWpf/Views/Core/ConnectorPinView.xaml

```diff
@@ -68,7 +68,7 @@
             </Image>
         </Grid>
         <Canvas.RenderTransform>
-            <TranslateTransform X="{Binding HalfWidth, Mode=OneWay}" Y="{Binding HalfWidth, Mode=OneWay}" />
```

Comment:
Just curious do we need to remove this `HalfWidth` property or?

Reply:
One sec having a look.